sdk/src/store.rs

```diff
@@ -5533,2991 +5779,2928 @@ pub mod tests {
 
-        let mut report = StatusTracker::with_error_behavior(ErrorBehavior::StopOnFirstError);
+            let mut um_report = StatusTracker::with_error_behavior(ErrorBehavior::StopOnFirstError);
 
-        // save to output
-        let mut store = Store::load_from_asset(ap.as_path(), true, &mut report).unwrap();
-        let pc = store.provenance_claim().unwrap();
-
-        // create a new update manifest
-        let mut claim = Claim::new("adobe unit test", Some("update_manifest_vendor"), 2);
-        // ClaimGeneratorInfo is mandatory in Claim V2
-        let cgi = ClaimGeneratorInfo::new("claim_v2_unit_test");
-        claim.add_claim_generator_info(cgi);
-
-        let ingredient_hashes = store.get_manifest_box_hashes(pc);
-        let parent_hashed_uri = HashedUri::new(
-            store.provenance_path().unwrap(),
-            Some(pc.alg().to_string()),
-            &ingredient_hashes.manifest_box_hash,
-        );
-        let signature_hashed_uri = HashedUri::new(
-            to_signature_uri(pc.label()),
-            Some(pc.alg().to_string()),
-            &ingredient_hashes.signature_box_hash,
-        );
+            // read back in store with update manifest
+            let mut um_store = Store::load_from_asset(op.as_path(), true, &mut um_report).unwrap();
 
-        let validation_results = ValidationResults::from_store(&store, &report);
+            let um = um_store.provenance_claim().unwrap();
 
-        let ingredient = Ingredient::new_v3(Relationship::ParentOf)
-            .set_active_manifests_and_signature_from_hashed_uri(
-                Some(parent_hashed_uri),
-                Some(signature_hashed_uri),
-            ) // mandatory for v3
-            .set_validation_results(Some(validation_results)); // mandatory for v3
+            // should be an update manifest
+            assert!(um.update_manifest());
 
-        claim.add_assertion(&ingredient).unwrap();
+            // should have valid bmff hash binding
+            assert!(um_report.has_status(validation_status::ASSERTION_BMFFHASH_MATCH));
 
-        // create mandatory opened action (optional for update manifest)
-        let ingredient = claim.ingredient_assertions()[0];
-        let ingregient_uri = to_assertion_uri(claim.label(), &ingredient.label());
-        let ingredient_hashed_uri = HashedUri::new(
-            ingregient_uri,
-            Some(claim.alg().to_owned()),
-            ingredient.hash(),
-        );
+            // now add a new oridinary manifest to restore the original manifest (collapsed manifest)
 
-        let opened = Action::new("c2pa.opened")
-            .set_parameter("ingredients", vec![ingredient_hashed_uri])
-            .unwrap();
-        let em = Action::new("c2pa.edited.metadata");
-        let actions = Actions::new().add_action(opened).add_action(em);
+            // create a new ordinary claim
+            let mut claim2 = Claim::new("adobe unit test", Some("ordinary_manifest_vendor"), 2);
```

Comment:
I thought we were not supposed to mention "adobe" to be vendor-agnostic? (There are other occurrences)